lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -227,29 +219,33 @@ private function checkExecutedUnavailableMigrations(
         return true;
     }
 
-    private function getVersionNameFromAlias(string $versionAlias) : void
+    private function errorForAlias(string $versionAlias, bool $allowNoMigration) : int
     {
-        if ($versionAlias === 'first') {
-            $this->io->error('Already at first version.');
+        if (in_array($versionAlias, ['first', 'next', 'latest'], true) || substr($versionAlias, 0, 7) === 'current') {
```

Comment:
You can make this length independent with `strpos`:

```suggestion
        if (in_array($versionAlias, ['first', 'next', 'latest'], true) || strpos($versionAlias, 'current') === 0 ) {
```